docs/writing-helmfile.md

```diff
@@ -14,18 +14,18 @@ That is, the following example let `helmfile` fail when you have no `eventApi.re
 {{ .Values.eventApi.replicas | default 1 }}
 ```
 
-In case it isn't a mistake and you do want to allow missing keys, use the `getOrNil` template function:
+In case it isn't a mistake and you do want to allow missing keys, use the `get` template function:
 
 ```
-{{ .Values | getOrNil "eventApi.replicas" }}
+{{ .Values | get "eventApi.replicas" }}
 ```
 
 This result in printing `<no value` in your template, that may or may not result in a failure.
```

Comment:
@mumoshu Is this still true if using `get` instead of `getOrNil`?

Reply:
No. `{{ .Values | get "eventApi.replicas" }}` fails with an error when the value does not exist.

So I guess we'd better change the example above to `{{ .Values | get "eventApi.replicas" nil }}`

Reply:
Done